src/test/java/seedu/address/logic/parser/EditCommandParserTest.java

```diff
@@ -134,6 +134,7 @@ public void parse_oneFieldSpecified_success() {
         // name
         Index targetIndex = INDEX_THIRD_PERSON;
         String userInput = targetIndex.getOneBased() + NAME_DESC_AMY;
+        System.out.println(userInput);
```

Comment:
Was this left from debugging? Should we remove?

Reply:
I think so. oops.